apiaccess.py

```diff
@@ -17,7 +17,30 @@ def _remote_headers(auth_string, case_id, language_model=None):
     return headers
 
 
-def call_endpoint(endpoint, auth_string, request_spec, case_id,
+def _flatten_params_dict(to_flatten, parent_key=''):
+    """Returns flattened dict with concatenated keys separated by dot
+
+    Args:
+        to_flatten (dict | None)
+
+    Returns:
+        dict | None: flattened dict, or None if given `to_flatten` is None
+    """
+    if to_flatten is None:
+        return None
+
+    items = []
```

Comment:
I think we can do `items = {}` and then use `items.update(…)` and `items[new_key] = value` without using intermediate list :)

Reply:
You were right :D I've changed it